rewrite-java/src/main/java/org/openrewrite/java/tree/JavaType.java

```diff
@@ -362,6 +401,23 @@ public boolean deepEquals(@Nullable JavaType type) {
         public String toString() {
             return "Class{" + fullyQualifiedName + '}';
         }
+
+        public enum Kind {
+
+            Class(0),
+            Enum(0x4000),
+            Interface(0x0200),
+            Annotation(0x2200);
+
+            private final int bitMask;
```

Comment:
Consider deleting this, as kinds aren't being bit-or'd together into a set like they are for modifiers.